test/integration/dialects/postgres/dao.test.js

```diff
@@ -332,6 +332,31 @@ if (dialect.match(/^postgres/)) {
         });
       });
 
+      it('should not rename hstore fields', function() {
```

Comment:
This could be a unit test instead https://github.com/legomind/sequelize/blob/8e12f8e9148540d2ad782570ec37163529f6c01f/test/unit/sql/select.test.js